sys-fs/python-btrfs/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+	<maintainer type="person" proxied="yes">
+		<email>holger@applied-asynchrony.com</email>
+		<name>Holger Hoffstätte</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="examples">Install examples</flag>
```

Comment:
"examples" is already a global USE flag:
>profiles/use.desc:77:examples - Install examples, usually source code
so we can just drop this, as the local description isn't much more descriptive.

Reply:
Of course! Removed.